frontend/src/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import Drawer from "@material-ui/core/Drawer";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+import closeButton from "./closeButton.png";
+
+const useStyles = makeStyles({
+  drawer: {
+    backgroundColor: "#E5E5E5",
```

Comment:
use `theme.palette.secondary.main` with `const theme = useTheme()`

Reply:
it's not the same color as in the figma

Reply:
also instead of classes={{ paper: theme.palette.secondary.main }} can I do classes={{ paper: "secondary" }} ? Seems cleaner.